source/Panels.js

```diff
@@ -737,11 +738,21 @@
 		},
 
 		/**
-		* Returns `true` if any panels will move in the transition from `fromIndex` to
+		* Returns `true` if the panels should animate in the transition from `fromIndex` to
 		* `toIndex`.
 		*
 		* @private
 		*/
+		shouldAnimate: function () {
+			this._willMove = this._willMove || (this._willMove == null && (this.shouldArrange() && this.animate));
+			return this._willMove;
+		},
```

Comment:
@aarontam, what you have here should work just fine, but how would you feel about this instead:

```javascript
if (this._willMove == null) {
	return (this._willMove = this.animate && this.shouldArrange());
}
else {
	return this._willMove;
}
```

It's a bit more verbose, but I think it makes the logic a bit more transparent and shouldn't be any slower to execute (in fact I think should be faster, only because we check the simple boolean this.animate before running the more expensive this.shouldArrange()).